src/cls/rbd/cls_rbd.cc

```diff
@@ -134,6 +134,9 @@ cls_method_handle_t h_mirror_image_status_get;
 cls_method_handle_t h_mirror_image_status_list;
 cls_method_handle_t h_mirror_image_status_get_summary;
 cls_method_handle_t h_mirror_image_status_remove_down;
+cls_method_handle_t h_mirror_image_create_marker_get;
```

Comment:
Got it. That would mean `image::CreateRequest` need not set mirror image key(s) for non-primary -- all that's done in `BootstrapRequest`